package.json

```diff
@@ -53,7 +53,7 @@
   },
   "homepage": "https://github.com/GetJobber/atlantis#readme",
   "devDependencies": {
-    "@apollo/client": "^3.7.10",
+    "@apollo/client": "^3.7.3",
```

Comment:
don't really need to downgrade this, no? Like the peer deps said 3.7.3 and above? or are we saying only 3.7.3?

Reply:
No technically but I did so to keep it consistent with the peer dependency so we can actually verify that it will work within Atlantis instead of building something using a newer version that breaks on older versions. I can revert though

Reply:
but JO is 3.7.10

Reply:
also, tbf, this would eventually install 3.7.13 on other projects when npm starts deduping them

Reply:
@darryltec I can undo the change here. 
Looking at the following libraries it seems like they don't pin patch versions and will generally unpin (*) or have it set to a major version. See below:

1. [Reanimated](https://github.com/software-mansion/react-native-reanimated/blob/main/package.json#L86-L94)
2. [Flashlist](https://github.com/Shopify/flash-list/blob/main/package.json#L54-L57)
3. [Popper](https://github.com/floating-ui/react-popper/blob/master/package.json#L60-L63)
4. [React Navigation Native stack](https://github.com/react-navigation/react-navigation/blob/main/packages/native-stack/package.json#L59-L64)
5. [React Navigation drawer](https://github.com/react-navigation/react-navigation/blob/main/packages/drawer/package.json#L64-L71)
6. [React Navigation Stack](https://github.com/react-navigation/react-navigation/blob/main/packages/stack/package.json#L61-L67)
Thoughts on just having it be `^3.0.0` to match the others?

Reply:
That's fine by me!

Reply:
Sounds good, I will push the change right now

Reply:
@darryltec updated